tensortrade/env/default/rewards.py

```diff
@@ -6,7 +6,8 @@
 
 from tensortrade.env.generic import RewardScheme, TradingEnv
 from tensortrade.feed.core import Stream, DataFeed
-
+from tensortrade.env.generic.utils.running_stats import Welfords, PctChange
```

Comment:
Let's revert this file back to the way it was. We can discuss how to do the online computations later on because I would like to incorporate these changes in at some point.

Reply:
He said its 10x faster, are you putting this upgrade on pause for now?